applications/phoenix/main.tf

```diff
@@ -107,7 +107,6 @@ module "app" {
   queue_scale = 0
 
   with_redis = true
-  redis_type = var.environment == "production" ? "premium-1" : "hobby-dev"
```

Comment:
I removed this override because, based on our low storage utilization on these instances, the extra storage afforded by `premium-1` is giving us no benefit over what we get from `premium-0` (at half the cost)!